addons/api/tests/unit/models/base-test.js

```diff
@@ -78,20 +78,27 @@ module('Unit | Model | base', function (hooks) {
         attributes: {},
       },
     });
+    this.server.patch('/v1/users/1', () => {
+      assert.ok(true, 'Correctly scoped update record URL was requested.');
+      return {};
+    });
     const model = store.peekRecord('user', '1');
     assert.false(model.canSave);
     assert.true(model.cannotSave);
     // Should be able to save if dirty
     model.name = 'User';
+
     assert.true(model.hasDirtyAttributes);
     assert.true(model.canSave);
     assert.false(model.cannotSave);
     // Should not be able to save while currently saving
-    model.transitionTo('updated.inFlight');
+    model.save().then(() => {
```

Comment:
I actually _don't_ want to await it. This is so I can assert the `isSaving` transition and then once the promise is resolved assert that it transitioned correctly.

Reply:
Ah got it.  There may be a way to express this in a way that accomplishes both goals.

Reply:
Made a suggestion.  I wanted to make the await explicit for readability.  While it probably works without await, that's because Ember tests automagically await the runloop.  It's perfectly valid, but might be confusing for anyone new to Ember.